docs/pages/management/dynamic-configuration.mdx

```diff
@@ -0,0 +1,238 @@
+---
+title: Using Dynamic Configuration Resources
+description: An introduction to Teleport's dynamic configuration resources, which make it possible to apply settings to remote clusters using infrastructure as code.
+tocDepth: 3
+---
+
+This section includes guides for managing Teleport's dynamic configuration
+resources, which make it possible to adjust the behavior of your Teleport
+cluster as your infrastructure changes.
+
+## What is a dynamic configuration resource?
```

Comment:
We should mention and link to the app and database dynamic configuration guides.